src/pages/staticPages/tools/oncoprinter/OncoprinterGeneticUtils.ts

```diff
@@ -217,7 +217,7 @@ export function getGeneSymbols(
 export async function fetchOncoKbDataForMutations(
     annotatedGenes: { [entrezGeneId: number]: boolean } | Error,
     data: OncoprinterGeneticTrackDatum_Data[],
-    client: OncoKbAPI = oncokbClient
+    client = oncokbClient
```

Comment:
I didn't know that it was possible to leave the type out when there is a default value.